src/components/SearchContainer/SearchContainer.css

```diff
@@ -0,0 +1,48 @@
+.search-container {
+  position: relative;
+  width: 100%;
+  height: 100%;
+  background-image: url("https://s3-alpha-sig.figma.com/img/9ab1/b611/92a81bbe79a924292111ca43f3a39e8f?Expires=1730073600&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=noXbQ~iaCuYPkcYdiT3A19bd~XpmS7lRSoTcHBmiSkegteEULSC00Du4EhN9oKvV4Dovbx50zzvM6ANyYqESQuM8Nqg4WEoNkn1GgNB~MHvokRJUWV9cLz8WUWmjwuprp2BmSm82OUq1e2RTSk0H2rWf2gTkGzCN4enHjNCwWooBhP5mn85oq9gB3m3ft6GSBYYBIg1zmjkf3JXU7H9zkR70km06FzPYRHdiB485lalfi3cJLneg8X1etHMAHkQsmba3~0oJtVOmEqeCgF8j~VGQ9fMjpWEgRhTjtucz7DGXcQH3YvuGhZexoGdZulYCu1UtxQR9he10zYyhtzpmRA__");
```

Comment:
I'm pretty sure that - this url will not work

Reply:
fixed